tests/controller/insertcontent.js

```diff
@@ -62,6 +62,57 @@ describe( 'DataController', () => {
 			expect( getData( doc ) ).to.equal( 'xa[]x' );
 		} );
 
+		describe( 'saves the reference to the original object', () => {
+			it( 'inline-widget', () => {
```

Comment:
Ok, If this enhancement requires only one test, I can change it to your proposal. But take a look at the whole file. It uses a little bit different naming in descriptions.